src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryRegEx.cs

```diff
@@ -12,7 +12,9 @@
 using Microsoft.PowerFx.Core.Texl.Builtins;
 using Microsoft.PowerFx.Core.Types;
 using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Interpreter.Localization;
 using Microsoft.PowerFx.Types;
+using static Microsoft.PowerFx.Syntax.PrettyPrintVisitor;
```

Comment:
>Microsoft.PowerFx.Syntax.PrettyPrintVisitor [](http://example.com/codeflow?start=13&length=43)

Why do we need this?

Reply:
We dont. Must had been auto added while coding.